bot.py

```diff
@@ -109,6 +109,7 @@
     'you ' : 'uwu ',
     'dude': 'duwude',
     'to' : 'towo',
+    'oh' : 'owo',
```

Comment:
Okay this commit message is probably not the best, I forgot I added this and it wanted me to commit before merging so 🤷